s3/pkg/service/bucket.go

```diff
@@ -207,11 +214,9 @@ func (s *s3Service) PutBucketLifecycle(ctx context.Context, in *pb.PutBucketLife
 		return nil
 	}
 
-	_, _, err = CheckRights(ctx, bucket.TenantId)
+	_, _, _, err = CheckRights(ctx, bucket.TenantId)
 	if err != nil {
-		if err == ErrNoSuchKey {
```

Comment:
Is it better to return ErrBucketAccessForbidden here,  otherwise the user may create a new one with the same name, but it will failed because duplicate key.